src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -263,11 +265,25 @@ private InitCodeView buildInitCodeViewRequestObject(
         context,
         root.listInInitializationOrder(),
         ImmutableList.of(root),
-        subTrees(root, initCodeContext.sampleArgStrings()));
+        sampleFuncParams(root, initCodeContext.sampleArgStrings()));
   }
 
-  private List<InitCodeNode> subTrees(InitCodeNode root, List<String> paths) {
-    return paths.stream().map(path -> root.subTree(path)).collect(ImmutableList.toImmutableList());
+  private List<InitCodeNode> sampleFuncParams(InitCodeNode root, List<String> paths) {
```

Comment:
brief function comment for private methods?

Reply:
Could you  make this comment a bulleted list for clarity?:
```
If path is:
* foo: do foo
* bar: do baar
```

Reply:
Ah, it got screwed up by the formatter :(